oreorebot2/compose.yml

```
@@ -0,0 +1,6 @@
+services:
+  oreorebot2:
```

コンテナ名は `<プロジェクト名>_<サービス名>_<番号>` になるので普遍的な名前のほうが良さそう

```suggestion
  app:
```